api.js

```diff
@@ -0,0 +1,26 @@
+// projects
+export function getProjects(axios) {
+  return axios.$get('/project')
```

Comment:
chore, question (blocking): using plural nouns is preferred unless the resource is clearly a singular concept. Is this also discussed with the backend to use this style? Otherwise, we should follow conventions here.

Reply:
Same comment for all other functions in this file.